### DISCUSSION TITLE
After upgrading to version 3.5.0 Alpine throws errors

### DISCUSSION THREAD

**Discussion by pfrlv** [alpinejs/alpine#2323] [1. Help] [answered]
`Alpine Expression Error: func(...).catch is not a function` (Safari Version 14.0.3 (16610.4.3.1.7) / Chrome (Version 95.0.4638.69 (Official Build) (arm64))) — i see in the console on any Alpine call.
I used Vite for bundling, but if test with Mix (webpack) everything is fine

**Comment by SimoTod** [comment, marked as answer]
Target? It seems like you are transpiling the code for old browsers but Alpine needs async functions to work.

**Reply by pfrlv** [reply]
es2015... and yes ) with es2017 everything is correct. It worked. Thanks for the solution 😘
